src/ray/core_worker/core_worker.cc

```diff
@@ -2282,22 +2282,26 @@ void CoreWorker::YieldCurrentFiber(FiberEvent &event) {
 void CoreWorker::PlasmaCallback(SetResultCallback success,
                                 std::shared_ptr<RayObject> ray_object, ObjectID object_id,
                                 void *py_future) {
-  std::vector<std::shared_ptr<RayObject>> vec;
+  // This is the base case. A typical call stack for async plasma get looks like:
+  // GetAsync -> PlasmaCallback (ask Raylet) -> HandlePlasmaObjectReady ->
+  // plasma_arrived_callback -> GetAsync -> PlasmaCallback (this base case).
```

Comment:
Maybe a nit, but I don't really understand this comment (I'm not very familiar with the async code). It would be nice if the comment could be self-contained, so you don't need to reference other parts of the code to understand this part.